conf/supervisord.conf

```diff
@@ -6,24 +6,24 @@ logfile_maxbytes=0
 
 [program:h-beat]
 command=celery -b %(ENV_H_BROKER_URL)s -A h_periodic.h_beat beat --pidfile=h-celerybeat.pid --loglevel INFO
-stdout_logfile=NONE
-stderr_logfile=NONE
 stdout_events_enabled=true
 stderr_events_enabled=true
+stdout_logfile=NONE
+stderr_logfile=NONE
 
 [program:checkmate-beat]
 command=celery -b %(ENV_CHECKMATE_BROKER_URL)s -A h_periodic.checkmate_beat beat --pidfile=checkmate-celerybeat.pid --loglevel INFO
 stdout_events_enabled=true
 stderr_events_enabled=true
-stopsignal = KILL
-stopasgroup = true
```

Comment:
The cookiecutter has actually changed these two settings in h-periodic for the production `checkmate-beat` and `lms-beat` programs. This makes the `checkmate-beat` and `lms-beat` programs the same as the `h-beat` one. I believe this is correct: the `stopsignal` and `stopasgroup` settings are meant for dev not prod. And programs in prod are apparently supposed to have the `stdout_logfile=NONE` and `stderr_logfile=NONE` settings. These settings just turn off Supervisor's default behaviour of writing the programs logs to files (see [Supervisor's docs](http://supervisord.org/configuration.html)) which makes sense: we take the logs from stdout and stderr directly into Papertrail, we wouldn't read any log files created on the VM's filesystem.

Reply:
Yeah, probably copy based based nonsense?